### PR TITLE
Introduces no-mix constraint

### PR DESCRIPTION
**Pull request by davidrijsman**
Introduces no-mix constraint

### CODE REVIEW

**Review by merschformann — COMMENTED** [review]
Looks good. Still going a bit back and forth on whether we should prepend the `no_` everywhere or just omit it everywhere. Or, use the current mix. It's probably not too important. :sweat_smile:

**Review by merschformann — APPROVED** [review]

File: nextroute/schema/input.go [hunk 1]
```diff
@@ -203,6 +203,17 @@ type Stop struct {
 	ID string `json:"id,omitempty"`
 	// Location where the stop is.
 	Location Location `json:"location,omitempty"`
+	// MixingItems defines the items that are inserted or removed from the vehicle when visiting the stop.
+	MixingItems any `json:"mixing_items,omitempty"`
```
Comment (sebastian-quintero) [inline nextroute/schema/input.go:207]: Should this be of type `MixItem`?

File: nextroute/schema/input.go [hunk 2]
```diff
@@ -203,6 +203,17 @@ type Stop struct {
...
+}
+
+// MixItem is an item that is used to specify the type of mix.
+type MixItem struct {
+	// Name is the name of the mix item.
```
Comment (sebastian-quintero) [inline nextroute/schema/input.go:212]:
```suggestion
	// Name of the mix item.
```

File: nextroute/schema/output.go [hunk 3]
```diff
@@ -78,6 +78,8 @@ type PlannedStopOutput struct {
 	EarlyArrivalDuration int `json:"early_arrival_duration,omitempty"`
 	// LateArrivalDuration is the late arrival duration of the stop in seconds.
 	LateArrivalDuration int `json:"late_arrival_duration,omitempty"`
+	// MixItems is the mix items of the stop.
+	MixItems any `json:"mix_items,omitempty"`
```
Comment (sebastian-quintero) [inline nextroute/schema/output.go:82]: Same question regarding the type.

**Review by sebastian-quintero — APPROVED** [review]
Should the new `Stop` property be also available in `AlternateStop` and/or `StopDefaults`?

File: nextroute/schema/input.go (hunk 1 above)
Comment (davidrijsman) [reply nextroute/schema/input.go:207]: No there can be multiple items that can not be mixed on a vehicle

File: nextroute/schema/output.go (hunk 3 above)
Comment (davidrijsman) [reply nextroute/schema/output.go:82]: Same answer, there can be multiple items that can not be mixed

**Pull request merged**